app/components/pagination-navigation.js

```diff
@@ -0,0 +1,122 @@
+import Ember from 'ember';
+import { alias } from 'ember-computed-decorators';
+
+export default Ember.Component.extend({
+  tagName: 'nav',
+  classNames: ['pagination-navigation'],
+  @alias('collection.pagination') pagination: null,
+
+  outerWindow: Ember.computed('outer', function () {
+    return this.get('outer') || 1;
+  }),
+  innerWindow: Ember.computed('inner', function () {
+    return this.get('inner') || 2;
+  }),
+
+  currentPage: Ember.computed('pagination.offset', 'pagination.perPage', function () {
```

Comment:
I think that we could move this to the `PaginatedCollection` class, so here in the component you could just say `pagination.currentPage`